cocos/core/animation/wrap.ts

```diff
@@ -0,0 +1,74 @@
+import { WrapMode, WrapModeMask, WrappedInfo } from './types';
+
+export function wrap (
```

Comment:
I feel that this name may not be clear enough, `wrap` is too general, this function is used to calculate the time and period information in the corresponding wrap mode, so `calcWrappedInfo` , `calcWrappedCycle` or other similar names maybe better?